docs/design/windows-design.md

```diff
@@ -210,8 +210,17 @@ Kube-proxy userspace mode is configured to provide NodePort Service function. A
 "HNS Internal NIC" is provided to kube-proxy to configure Service addresses. The OpenFlow entries for the
 NodePort Service traffic on Windows are the same as those on Linux.
 
-The ClusterIP Service function is implemented by Antrea leveraging OVS. Antrea installs OpenFlow entries
-to select the backend endpoint and performs DNAT on the traffic.
+The ClusterIP Service function is implemented by AntreaProxy. Antrea installs routes from ClusterIP Services
```

Comment:
I feel you should add more details here. Like:

> Antrea Agent installs routes to send ClusterIP Service traffic from host network to the OVS bridge. For each Service, it adds a route that routes the traffic via a virtual IP (169.254.169.252), and it also add a route to indicate the virtual IP is reachable via antrea-gw0.

Reply:
Updated.